code/modules/mob/living/carbon/human/species_types/monkeys.dm

```diff
@@ -0,0 +1,92 @@
+/datum/species/monkey
+	name = "Monkey"
+	id = "monkey"
+	say_mod = "chimpers"
+	attack_verb = "bite"
+	attack_sound = 'sound/weapons/bite.ogg'
+	miss_sound = 'sound/weapons/bite.ogg'
+	mutant_organs = list(/obj/item/organ/tail/monkey)
+	mutant_bodyparts = list("tail_monkey" = "Monkey")
+	skinned_type = /obj/item/stack/sheet/animalhide/monkey
+	meat = /obj/item/food/meat/slab/monkey
+	species_traits = list(HAS_FLESH,HAS_BONE,NO_UNDERWEAR,LIPS,NOEYESPRITES,NOBLOODOVERLAY)
+	inherent_traits = list(TRAIT_MONKEYLIKE)
+	no_equip = list(ITEM_SLOT_EARS, ITEM_SLOT_EYES, ITEM_SLOT_OCLOTHING, ITEM_SLOT_GLOVES, ITEM_SLOT_FEET, ITEM_SLOT_ICLOTHING, ITEM_SLOT_SUITSTORE)
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
+	liked_food = MEAT | FRUIT
+	limbs_id = "monkey"
+	damage_overlay_type = "monkey"
+	sexes = FALSE
+	punchdamagelow = 1
+	punchdamagehigh = 3
+	punchstunthreshold = 4 // no stun punches
+	species_language_holder = /datum/language_holder/monkey
+	bodypart_overides = list(
+	BODY_ZONE_L_ARM = /obj/item/bodypart/l_arm/monkey,\
+	BODY_ZONE_R_ARM = new /obj/item/bodypart/r_arm/monkey,\
+	BODY_ZONE_HEAD = new /obj/item/bodypart/head/monkey,\
+	BODY_ZONE_L_LEG = new /obj/item/bodypart/l_leg/monkey,\
+	BODY_ZONE_R_LEG = new /obj/item/bodypart/r_leg/monkey,\
+	BODY_ZONE_CHEST = new /obj/item/bodypart/chest/monkey)
+	dust_anim = "dust-m"
+	gib_anim = "gibbed-m"
+
+	payday_modifier = 1.5
+
+
+/datum/species/monkey
+
+/datum/species/monkey/random_name(gender,unique,lastname)
+	var/randname = "monkey ([rand(1,999)])"
+
+	return randname
+
+/datum/species/monkey/on_species_gain(mob/living/carbon/C, datum/species/old_species)
+	. = ..()
+	C.ventcrawler = VENTCRAWLER_NUDE
+	C.pass_flags |= PASSTABLE
+	C.ai_controller = /datum/ai_controller/monkey
```

Comment:
What if there's already a controller there ?

Reply:
some leftover fikou code, lemme kill this it doesn't need to exist

Reply:
bitch i made this before ai controllers were a tihng..........